### PR TITLE
Query language update guide

### PR DESCRIPTION
**Pull request by jamesm-macrometa**
Hi @grainier-macrometa - I'm opening this PR to upload the Stream Workers V1 to V2 update guide.  Can you please take a look at the updates in this PR and verify that my descriptions are correct?

@elof - please wait until Grainier has approved this PR to merge and deploy.  Thanks!

### CODE REVIEW

File: docs/cep/reference/query-language-update.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
+# Query Language Syntax Update
+
+We updated the query language syntax for Stream QL applications to be more consistent with the industry standard. This section describes the differences between the updated syntax (version 2) and the old syntax (version 1), and how to update your applications from version 1 to version 2. Updating to version 2 is not necessary as version 1 applications are still supported. Applications provided by Macrometa are automatically updated.
+
+You must include `@App:version('2')` at the beginning of the stream application so Stream QL recognizes it as version 2.
```
Comment (grainier-macrometa) [inline docs/cep/reference/query-language-update.md:5]: @jamesm-macrometa - `version` was recently changed to `qlVersion`. So this should be `@App:qlVersion('2')`.

File: docs/cep/reference/query-language-update.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
+# Query Language Syntax Update
+
+We updated the query language syntax for Stream QL applications to be more consistent with the industry standard. This section describes the differences between the updated syntax (version 2) and the old syntax (version 1), and how to update your applications from version 1 to version 2. Updating to version 2 is not necessary as version 1 applications are still supported. Applications provided by Macrometa are automatically updated.
+
+You must include `@App:version('2')` at the beginning of the stream application so Stream QL recognizes it as version 2.
+
+For all queries, version 2 replaces annotations such as `@sink` and `@source` with `WITH()` properties. For example:
```
Comment (grainier-macrometa) [inline docs/cep/reference/query-language-update.md:7]: Annotations were replaced by `Keywords` such as `SINK`and `SOURCE` and `WITH()` properties.

cc: @jamesm-macrometa

File: docs/cep/reference/query-language-update.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+    CREATE SOURCE STREAM ExampleStreamSource (data string);
+
+    // Creates GDN stream sink:
+    CREATE SINK STREAM ExampleStreamSink (data string);
+    ```
+
+
+### Define Tables
+
+Tables and collections also no longer require `@sink` or `@source` annotations.
```
Comment (grainier-macrometa) [inline docs/cep/reference/query-language-update.md:87]: Only `GDN` tables don't require `@sink` and `@source` annotations.

File: docs/cep/reference/query-language-update.md [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+
+* Version 1:
+
+    ```
+    define table ExampleTable (data string);
```
Comment (grainier-macrometa) [inline docs/cep/reference/query-language-update.md:92]: This should be;
```sql
@store(type=’c8db’, collection=`<table_name>`)
define table <table_name> (<attribute_name> <attribute_type>, ...);
```

cc: @jamesm-macrometa

**Review by grainier-macrometa — APPROVED** [review]
@jamesm-macrometa I think I've made some mistakes in the guide. I've fixed them and added comments here. Please fix them.  Other than that, LGTM.

**Comment by jamesm-macrometa** [comment]
Thanks @grainier-macrometa - made your listed changes in the doc.

@elof This content is now approved and ready to go at your convenience.  🙂

**Pull request merged**